python_modules/dagster/dagster/_core/storage/event_log/sqlite/sqlite_event_log.py

```diff
@@ -436,14 +437,6 @@ def fetch_run_status_changes(
     def supports_event_consumer_queries(self) -> bool:
         return False
 
-    def delete_events(self, run_id: str) -> None:
```

Comment:
confirming understanding: this is because it is now possible to use the base class implementation

Reply:
that's right.